Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -768,10 +768,7 @@ namespace Babylon
         const auto& vertexArray = *(info[0].As<Napi::External<VertexArray>>().Data());
 
         // a vertex array might not have an index buffer associated with
```

Comment:
this comment doesn't make sense now

Reply:
It does! the m_currentBoundIndexBuffer is updated with the index buffer associated with it. It can be nullptr.

Reply:
Oh, right, sorry, I misinterpreted the comment.